templates/taxi/manufacturer_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Deleting manufacturer {{ manufacturer.name }}</h1>
+  <br>
+  <h5><i>Are you sure you want to delete this manufacturer?</i></h5>
+
+  <form action="" method="post">
+  {% csrf_token %}
+    <input type="submit" value="Yes, submit" class="btn btn-danger">
+  </form>
+{% endblock %}

```

Comment:
Can you add a "No" or "Cancel" button? This is not a mistake, but it looks nicer, more complete